docs/code/common.rst

```diff
@@ -16,6 +16,48 @@ Exceptions
 .. autoclass:: forte.common.EntryNotFoundError
     :members:
 
+.. autoclass:: forte.common.ProcessorConfigError
+    :members:
+
+.. autoclass:: forte.common.PackDataException
+    :members:
+
+.. autoclass:: forte.common.ProcessFlowException
+    :members:
+
+.. autoclass:: forte.common.ProcessExecutionException
+    :members:
+
+.. autoclass:: forte.common.ValidationError
+    :members:
+
+.. autoclass:: forte.common.ResourceError
+    :members:
+
+.. autoclass:: forte.common.ExpectedRecordNotFound
+    :members:
+
+.. autoclass:: forte.common.InvalidOperationException
+    :members:
+
+.. autoclass:: forte.common.UnknownOntologyClassException
+    :members:
+
+
+
+
+Configuration
+==========
```

Comment:
These lines need to be at least the same length as that the title string (i.e. `Configuration`), same below